samples/client/petstore/java/okhttp-gson/src/main/java/org/openapitools/client/ApiClient.java

```diff
@@ -243,7 +241,6 @@ public ApiClient(String basePath, String clientId, String clientSecret, Map<Stri
         authentications.put("api_key_query", new ApiKeyAuth("query", "api_key_query"));
         authentications.put("http_basic_test", new HttpBasicAuth());
         authentications.put("bearer_test", new HttpBearerAuth("bearer"));
-        authentications.put("http_signature_test", new HttpBearerAuth("signature"));
```

Comment:
note for discussion: when http_signature_test is kept, the samples contain this incorrect line of code